gunicorn/arbiter.py

```diff
@@ -583,6 +583,16 @@ def manage_workers(self):
                                   "value": active_worker_count,
                                   "mtype": "gauge"})
 
+        if self.cfg.enable_backlog_metric:
+            backlog = sum(sock.get_backlog() or 0
+                          for sock in self.LISTENERS)
+
+            if backlog:
```

Comment:
this will be always true there.even when backlog is set to -1.
I think correct test is :`if backlog >= 0'

Reply:
Ah yes, I missed this - it should be `>` because backlog can be `0`